cobra/flux_analysis/variability.py

```diff
@@ -1,12 +1,74 @@
 from warnings import warn
 
 from six import iteritems
-from ..solvers import solver_dict, get_solver_name
+from cobra.solvers import solver_dict, get_solver_name
+import cobra.util.solver as sutil
+from sympy.core.singleton import S
 
 
-def flux_variability_analysis(cobra_model, reaction_list=None,
+def flux_variability_analysis(model, reaction_list=None,
                               fraction_of_optimum=1.0, solver=None,
-                              objective_sense="maximize", **solver_args):
+                              **solver_args):
+    """Runs flux variability analysis to find the min/max flux values for each
+    each reaction in `reaction_list`.
+
+    Parameters
+    ----------
+    model : a cobra model
+        The model for which to run the analysis. It will *not* be modified.
+    reaction_list: list of cobra.Reaction or str, optional
```

Comment:
Yeah sure. That was a typo. I try to use the space before ":" but sometimes muscle memory gets in the way.